### ISSUE TITLE
[Request] Tanári oldal

### ISSUE THREAD

**Issue by merenyimiklos** [refilc/naplo#134]
Azt szeretném kérdezni, hogy én mint tanár is használhatom ezt a felületet?
Próbálkoztam belépni, de nem engedett..

Nagyon szuper fejlesztésnek gondolom ezt én, mint fejlesztő + programozás oktató és szeretném kipróbálni, de valamiért nem enged be a felhasználómmal.

Esetleg valakivel fel tudnám venni a kapcsolatot ez ügyben? (lenne még + kérdéseim is)

**Comment by doomhyena** [comment]
Szia! Nem vagyok fejlesztő habár tudok válaszolni. Sajnos nem, nem tudod használni mint tanár, nincs tanári verziója sajnos a Refilcnek. A fejlesztőknek nincsnen tanári fiókjuk így nincs lehetőségük megoldani, hogy a tanári részt is lefejlesszék.

**Comment by merenyimiklos** [comment]
És ha esetleg egy collaborator segítene ebben, mint tanár? :) 
Nagyon szívesen benne lennék!
Csak kellene pár instruction, hogy mit hogyan kellene. (Flutter-t már fejlesztettem)

Közben itt egy kérdést fel is tennék, van e olyan API a krétának, amivel azt meg lehet csinálni, hogy a helyettesítést lássuk?

**Comment by PPearoo** [comment]
Hali, nem fejlesztek de a projektnek része vagyok.

Contribnak örülünk mindig, ha lecloneozod a projektet és valahogy meg tudod babrálni hogy működjön tanári fiókkal, az nagyon jó lenne. De ahogy én láttam néhány tanári fiókot, gyökeresen meg kéne változtatni a funkcionalitását az alkalmazásnak, a tanári fiókok működéséhez. Szerintem max az órarendet lehetne egyelőre megoldani. De egyébként nem egy rossz ötlet, csinálhatnánk tanári alkalmazást is, csak egyikünk sem tanár (még.)

**Comment by merenyimiklos** [comment]
@PPearoo esetleg valamikor tudnánk beszélni?
Én szerintem nagyon jó lenne a tanári funkció hozzá! Én teljes mértékben benne lennék!

**Comment by PPearoo** [comment]
Persze, jelölj be discordon és konzultálok a többiekkel. Nevem: pearoo
